src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/DateAdd_TimeZone_Beirut.txt

```diff
@@ -0,0 +1,69 @@
+﻿// (UTC+02:00) Beirut
+#SETUP: TimeZoneInfo("Middle East Standard Time")
+
+>> DateAdd(Date(2021, 3, 26), 2)
+DateTime(2021,3,28,1,0,0,0)
+
```

Comment:
This is the same case as the one below, we probably don't need it

Reply:
We know they are the same but I think it's still valuable to check they are producing the same result